include/contractor/graph_contractor_adaptors.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef OSRM_CONTRACTOR_GRAPH_CONTRACTION_ADAPTORS_HPP_
+#define OSRM_CONTRACTOR_GRAPH_CONTRACTION_ADAPTORS_HPP_
+
+#include "contractor/contractor_graph.hpp"
+#include "util/log.hpp"
+
+#include <vector>
+
+namespace osrm
+{
+namespace contractor
+{
+
+// Make sure to move in the input edge list!
+template <typename InputEdgeContainer>
+std::vector<ContractorEdge> ContractorEdgeInputAdaptor(InputEdgeContainer input_edge_list)
```

Comment:
We use lower-case first letter for free functions. Should probably be named `toContractorEdge`, or at least something that is not a noun which is usually reserved for classes.